gradle.properties

```diff
@@ -0,0 +1,14 @@
+org.gradle.caching=true
+group=io.ballerina.lib
+version=1.0.0-SNAPSHOT
+ballerinaLangVersion=2201.12.0
+
+shadowJarPluginVersion=8.1.1
+downloadPluginVersion=5.4.0
+releasePluginVersion=2.8.0
+ballerinaGradlePluginVersion=3.0.0
```

Comment:
I think for the standard github workflow files, the supported gradle plugin version is `2.3.0`

Reply:
This is ok. since we don't have any native code